x-pack/plugins/cloud_security_posture/server/create_indices/create_indices.ts

```diff
@@ -140,6 +151,64 @@ const createLatestFindingsIndex = async (esClient: ElasticsearchClient, logger:
   }
 };
 
+const createLatestVulnerabilitiesIndex = async (esClient: ElasticsearchClient, logger: Logger) => {
+  try {
+    // Deletes old assets from previous versions as part of upgrade process
+    const INDEX_TEMPLATE_V830 = 'cloud_security_posture.vulnerabilities_latest';
+    await deleteIndexTemplateSafe(esClient, logger, INDEX_TEMPLATE_V830);
```

Comment:
AFAIK there aren't any older index templates to delete.
In that case, it is redundant